funnel/forms/project.py

```diff
@@ -63,8 +63,8 @@ class ProjectForm(forms.Form):
         description=__(u"Eg: Explara, Instamojo"),
         validators=[forms.validators.Optional(), forms.validators.Length(max=2000)])
 
-    def validate_date_upto(self, date_upto):
-        if self.date_upto.data < self.date.data:
+    def validate_date_upto(self, field):
+        if field.data < self.date.data:
             raise forms.ValidationError(_("End date cannot be before start date"))
```

Comment:
Use the `GreaterThan` validator. https://github.com/hasgeek/baseframe/blob/master/baseframe/forms/validators.py#L80-L94

Reply:
Sorry, `LesserThan` here.

Reply:
Validator on `date_upto` will be `GreaterThan('date')`